src/main/java/com/targomo/client/api/StatisticTravelOptions.java

```diff
@@ -310,4 +313,11 @@ public String toString(){
         return builder.toString();
     }
 
+    public Boolean getMultiGraphIgnoreRoutingErrorMessages() {
+        return multiGraphIgnoreRoutingErrorMessages;
+    }
+
+    public void setMultiGraphIgnoreRoutingErrorMessages(Boolean multiGraphIgnoreRoutingErrorMessages) {
+        this.multiGraphIgnoreRoutingErrorMessages = multiGraphIgnoreRoutingErrorMessages;
+    }
```

Comment:
I think your change is not sufficient you need to change the parser class also :thinking:

Reply:
will check in the Stat service. 
There is a "JacksonRequestConfigurator" present there which is doing direct conversion.